invenio_vocabularies/contrib/names/jsonschemas/names/name-v1.0.0.json

```diff
@@ -15,15 +15,41 @@
       "description": "Identifier of the name scheme.",
       "$ref": "local://definitions-v1.0.0.json#/identifier"
     },
-    "name": {"type": "string"},
-    "given_name": {"type": "string"},
-    "family_name": {"type": "string"},
+    "name": {
+      "type": "string"
+    },
+    "given_name": {
+      "type": "string"
+    },
+    "family_name": {
+      "type": "string"
+    },
     "identifiers": {
       "description": "Identifiers for the person.",
       "type": "array",
-      "items": {"$ref": "local://definitions-v1.0.0.json#/identifiers_with_scheme"},
+      "items": {
+        "$ref": "local://definitions-v1.0.0.json#/identifiers_with_scheme"
+      },
       "uniqueItems": true
     },
+    "props": {
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "boolean"
+            },
+            {
+              "type": "array"
```

Comment:
Is array wanted and maybe copied from somewhere else? It might be difficult to have good search queries with arrays.

Reply:
I thought to use to store affiliations at the beginning but not anymore needed as we only store them at the record level not in the props, I will remove it 👍